controllers/csm_controller.go

```diff
@@ -328,10 +328,8 @@ func (r *ContainerStorageModuleReconciler) Reconcile(ctx context.Context, req ct
 	}
 
 	newStatus := csm.GetCSMStatus()
-	requeue, err := utils.HandleSuccess(ctx, csm, r, newStatus, oldStatus)
-	if err != nil {
```

Comment:
why are we getting rid of this error message? I'm good with making it a warning instead of an error, but I don't think we should get rid of it.

Reply:
@AkshaySainiDell ^